spec/jobs/google_analytics_job_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe GoogleAnalyticsJob do
+  describe '.perform' do
+    it 'calls Staccato tracker' do
+      tracker = instance_double(Staccato::NoopTracker)
+      allow(Staccato).to receive(:tracker).with(nil, nil, ssl: true).and_return(tracker)
+      
+      expect(tracker).to receive(:event).with(action: '1234')
+
+      GoogleAnalyticsJob.perform_now(event_name: '1234')
```

Comment:
maybe we should test `perform_later` since that is the method we are using in `Analytics` ? I think that because we have `config.active_job.queue_adapter = :test` that would still run the job immediately so we could keep the same expectations here